viewer/packages/api/src/public/migration/Cognite3DViewer.ts

```diff
@@ -407,6 +411,7 @@ export class Cognite3DViewer {
     }
 
     this.spinner.dispose();
+    this.sessionLogger.dispose();
```

Comment:
Can we be so sure this will be called at the end of every session? It seems very dependent on someone actually calling the `dispose` method on the viewer, which I'd guess doesn't always happen. But then again, I don't know what else we would do :thinking:

Reply:
Discussed it with Lars, specifically for that reason there is a check for canvas visibility in `SessionLogger`. So, if someone forgot to dispose the viewer, `sessionEnded` events won't be fired since canvas will be invisible.